src/frontend/screens/Settings/MapSettings/BackgroundMaps.tsx

```diff
@@ -226,6 +231,10 @@ export const BackgroundMaps: NativeNavigationComponent<"BackgroundMaps"> = ({
 
 BackgroundMaps.navTitle = m.BackgroundMapTitle;
 
+export function convertBytesToMb(bytes: number) {
+  return (bytes / 1024 ** 2).toFixed(0);
```

Comment:
i'd be careful about the including the `toFixed` here, since it then returns a string and also is prone to loss of precision. would remove that part and let the caller of this function decide what to do with the numerical result

Reply:
the ui specifically rounds to the nearest whole number, hence the toFixed. But the function name is a little misleading I suppose

Reply:
think this helper is more useful without prescribing the precision, so i'd lean towards calling toFixed wherever this function is being used to offer the flexibility for other potential uses